qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsIndexModeRollingUpgradeIT.java

```diff
@@ -91,39 +93,6 @@ protected String getTestRestCluster() {
           }
         }""";
 
-    private static final String LOGS_TEMPLATE = """
```

Comment:
@salvatore-campagna fyi, this test both used cluster.logsdb.enabled=true and template with index.mode set to logsdb. With this change because cluster.logsdb.enabled was set to true, the data stream was in logsdb mode from  the start. So I slightly changed this test, so that we still test rolling upgrade from standard mode and after upgrade update to logsdb.

Reply:
Now that the setting is dynamic we can do that...before it was static...